client/state/ui/first-view/selectors.js

```diff
@@ -23,9 +19,9 @@ import { getCurrentUserDate } from 'state/current-user/selectors';
 export function getConfigForCurrentView( state ) {
 	const currentRoute = last( filter( getActionLog( state ), { type: ROUTE_SET } ) );
 	const path = currentRoute.path ? currentRoute.path : '';
-	const config = FIRST_VIEW_CONFIG.filter( entry => some( entry.paths, entryPath => startsWith( path, entryPath ) ) );
+	const config = find( FIRST_VIEW_CONFIG, ( entry => some( entry.paths, entryPath => startsWith( path, entryPath ) ) ) );
```

Comment:
>findLast perhaps

@jonathansadowski Would we ever expect the filter to have matched more than one item here? (re: #7284)